deploy/examples/cluster-stretched-aws.yaml

```diff
@@ -126,10 +126,10 @@ spec:
   placement:
     # The arbiter mon can have its own placement settings that will be different from the mons.
     # If the arbiter section is not included in the placement, the arbiter will use the same placement
-    # settings as other mons. In this example, the arbiter has a toleration to run on a master node.
+    # settings as other mons. In this example, the arbiter has a toleration to run on a control-plane node.
     arbiter:
       tolerations:
-        - key: node-role.kubernetes.io/master
+        - key: node-role.kubernetes.io/control-plane
```

Comment:
As we need to support the older kubernetes version, the Previous version of Kubernetes will have this label?

Reply:
@travisn 1.24 onwards the new taint and label will only available. I have added a comment on the configuation yaml for making it clear.. hope it convey.